src/lib/ajvInstance.ts

```diff
@@ -37,14 +42,34 @@ export const validationErrorToString = async (errors: AjvErrors) => {
   return (
     "Errors:\n\n" +
     errors
-      .filter(({ instancePath }) => {
-        return instancePath !== "";
-      })
-      .map(({ instancePath, params, message }) => {
+      .map(({ instancePath, params, message }, i) => {
         const paramsMessage = yamlDiffPatch("", {}, params);
-        return `${color.yellow(instancePath)} ${
+        const prev = errors[i - 1];
+
+        const isPreviousVersionError =
+          latestConfigVersion !== undefined &&
+          instancePath === "/version" &&
+          message === "must be equal to constant" &&
+          !paramsMessage.includes(latestConfigVersion);
+
+        const isDuplicateError =
+          prev?.instancePath === instancePath &&
+          jsonStringify(prev.params) === jsonStringify(params) &&
+          prev.message === message;
+
+        if (isPreviousVersionError || isDuplicateError) {
+          return "";
+        }
+
+        return `${instancePath === "" ? "" : `${color.yellow(instancePath)} `}${
           message ?? ""
-        }\n${paramsMessage}`;
+        }${paramsMessage === "" ? "" : `\n${paramsMessage}`}`;
+      })
+      .filter((s) => {
+        return (
+          s !== "" &&
+          s !== "must match exactly one schema in oneOf\npassingSchemas: null\n"
+        );
```

Comment:
If the error changes then user will start seeing a new useless error, I am removing it here only because it's useless. I don't think moving it to const will help, but probably writing a test could, but I think writing a test for this would be too much, this bit is not so important

Reply:
Can move it to const just for little bit of documentation purpose

Reply:
But can you control this error on ajv? E.g. intercept it before error reaches the user. For example in js-client i use zod which allows me to intercept errors by filtering their codes and classes, change their message and even silence them.
[Zod error API](https://zod.dev/ERROR_HANDLING).
Because if i understand the code correctly, you want to prevent certain kind of errors from getting in user console.

Reply:
Maybe you can give it a try soon or later with TODO and separate task. Dunno how hard would be to rewrite such places (if it even possible).